program-runtime/src/compute_budget.rs

```diff
@@ -674,6 +692,115 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_set_compute_unit_price_instruction() {
```

Comment:
Yea agree, I did it this way to fit in existing tests' style, which seems to group tests by functions; this way of organizing  helps me to visualize test scenarios/coverage better and maintain test code bit easier (since all relevant test codes are in one place) . But I don't feel too strong about this. I'll be happy to make change, if you push one more time 😃